### DISCUSSION TITLE
multi selection like xlsx

### DISCUSSION THREAD

**Discussion by Alkimay** [nick-keller/react-datasheet-grid#207] [Feature requests]
hello do you have any idea how to implement multi select for example selecting squere of 4x4 cells with shift.
the pressing ctrl on other unselected cell and its add it to selection if pressing shift now it will add selection to other according to this cell

**Comment by nick-keller** [comment]
Hey @Alkimay ,
This is not possible, and not planned.
It's just too much complexity for a very low added value :/
